SISRP-6395 - Add custom css class to control page width & margin

marcpechaitis opened the pull request:
https://jira.berkeley.edu/browse/SISRP-6395

marcpechaitis committed:
SISRP-6395 - Add custom css class to control page width & margin

https://jira.berkeley.edu/browse/SISRP-6395

christianvuerings commented on src/css/cs/items/_kitchensink.scss:
```diff
@@ -7,6 +7,11 @@
 
+  .uc-page {
+    max-width: 1000px;
+    margin: 0 auto;
```
@marcpechaitis  looks like `scss-lint` is failing, just putting `max-width` after `margin` should fix that. Could you push up a fix?

marcpechaitis committed:
SISRP-6395 - Add custom css class to control page width & margin

https://jira.berkeley.edu/browse/SISRP-6395

changed order, put “max-width” after “margin”

marcpechaitis commented:
@christianv I have made the change we discussed, moved "max-width" after "margin"
